client/src/app/actions/authenticate.js

```diff
@@ -1,94 +1,118 @@
-import jwtdecode from 'jwt-decode';
+import {showErrorNotification, showSuccessNotification} from './notifications';
+import {Toast} from 'react-materialize';
 
 import { USER_LOGGED_IN,
   USER_LOGGED_OUT,
-  USER_SIGN_IN_FAILURE,
   SIGNUP_USER_SUCCESS,
-  SET_CURRENT_USER } from './actiontype';
+  USER_AUTH_FAILURE
+} from './actiontype';
 
 import api from './api';
 import setAuthorizationToken from '../utils/setAuthorizationToken';
 
-export const setCurrentUser = user =>
-  ({
-    type: SET_CURRENT_USER,
-    user
-  });
+
+
+/**
+ * create action: userLoggedIn: user
+ * @function userLoggedIn
+ * @param {object} response
+ * @returns {object} action: type and response
+ */
 export const userLoggedIn = user =>
   ({
     type: USER_LOGGED_IN,
-    isAuthenticated: true,
     user
   });
+
+/**
+ * create action: userLoggedIn: user
+ * @function userLoggedOut
+ * @param {object} response
+ * @returns {object} action: type and response
+ */
 export const userLoggedOut = user =>
   ({
     type: USER_LOGGED_OUT,
-    isAuthenticated: false,
     user
   });
-export const signInUserFailure = error =>
-  ({
-    type: USER_SIGN_IN_FAILURE,
-    isAuthenticated: false,
-    error
-  });
-export const signUpUserSuccess = user => ({ type: SIGNUP_USER_SUCCESS, user });
 
 
 /**
- *
- * @param {*} data
- * @returns {*} user.data
+ * create action: userAuthFailure : user
+ * @function userAuthFailure
+ * @param {object} response
+ * @returns {object} action: type and response
+ */
+export const userAuthFailure = (user) =>
+({
+  type: USER_AUTH_FAILURE,
+  user
+});
+
```

Comment:
Take out lines 51 and 52. Note that you should have one line spacing between different functions or method anything more than one should be taken out.